model/task_queue_service_test.go

```diff
@@ -1724,20 +1509,11 @@ func (s *taskDAGDispatchServiceSuite) TestFindNextTaskForOutdatedHostAMI() {
 		Revision:            "1a53e026e05561c3efbb626185e155a7d1e4865d",
 		RevisionOrderNumber: 1846,
 	}
-	item1 := TaskQueueItem{
-		Id:            "1",
-		Group:         "",
-		BuildVariant:  "init_test_run",
-		Version:       "5d88953e2a60ed61eefe9561",
-		Project:       "ops-manager-kubernetes",
-		Requester:     "patch_request",
-		GroupMaxHosts: 0,
-		IsDispatched:  false,
-	}
 
 	t2 := task.Task{
 		Id:                  "2",
 		BuildId:             "ops_manager_kubernetes_e2e_openshift_cloud_qa_patch_1a53e026e05561c3efbb626185e155a7d1e4865d_5d88953e2a60ed61eefe9561_19_09_23_09_49_51",
+		ExecutionPlatform:   task.ExecutionPlatformHost,
```

Comment:
Is there a significance to adding this to the tests? I don't think it's important since it would be set in reality, but I'm mostly curious if it makes a difference to the test passing/failing.

Reply:
It was necessary in an earlier version of me refactoring this test because I was using this [query](https://github.com/evergreen-ci/evergreen/blob/155748ac4e9dd08b1cebcd67b045c998aef651a7/model/task/db.go#L682) and thought setting execution platform was necessary, but I'm not using this query anymore and on second look I didn't even need to explicitly set it in the first place. Removing.